### DISCUSSION TITLE
npm run start leading to UnhandledPromiseRejectionWarning

### DISCUSSION THREAD

**Discussion by iankm** [tailwindlabs/tailwindcss#4913] [Help]
Hey there!

I have been using tailwindcss for a few months but have run into an issue today with building my project. When I run `npm run start` I get this error:

<img width="1182" alt="Screen Shot 2021-07-06 at 9 31 09 PM" src="https://user-images.githubusercontent.com/10765530/124686770-ac7e6e80-dea1-11eb-9f82-3206b29fd790.png">

I know that mini-css-extract-plugin is a dependency of tailwindcss. I have tried uninstalling/reinstalling and I have gone through each of my packages to see if there was a circular dependency. However, I haven't found anything and none of my potential solutions have worked.

Here are my devDependecies:
<img width="343" alt="Screen Shot 2021-07-06 at 9 34 25 PM" src="https://user-images.githubusercontent.com/10765530/124686922-f5362780-dea1-11eb-8ad4-34581216214b.png">

My node version is `v14.15.2`

Any help on this would be greatly appreciated. Thank you.

**Comment by denik1981** [comment]
I believe the assumption of mini-css-extract-plugin being a dependency of tailwindcss isn't correct. 
At least for the latest version of TW I haven't found that package under the full in-depth dependency tree.
